pinot-common/src/main/java/com/linkedin/pinot/pql/parsers/pql2/ast/FunctionCallAstNode.java

```diff
@@ -71,6 +76,7 @@ public AggregationInfo buildAggregationInfo() {
     AggregationInfo aggregationInfo = new AggregationInfo();
     aggregationInfo.setAggregationType(function);
     aggregationInfo.putToAggregationParams("column", identifier);
+    aggregationInfo.setIsItInSelectList(_isInSelectList);
```

Comment:
Should be either `isInSelectList` (preferred) or `isItInSelectList`, but it should be consistent everywhere.